### DISCUSSION TITLE
Charts Dynamic Update

### DISCUSSION THREAD

**Discussion by andremene182** [N1ghtly/react-native-responsive-linechart#50] [Q&A]
Hi,
could charts being updated (re-rendered) passing the react state as data?

I need this for creating dynamic data charts.

Thanks :)

**Comment by xanderdeseyn** [comment]
Hi @andremene182, I believe the chart should update when you change the data prop? Is this not the case for you?

**Reply by nickyiie** [reply]
Hey there, I am experiencing the same issue. When I try to change the data prop using state I receive the following message : 
<img width="401" alt="image" src="https://user-images.githubusercontent.com/64499552/126387742-4a2816b2-2ffc-4eb9-89f3-0fa811d064e7.png">

**Reply by pillaedu** [reply]
Hello, I am getting the same issue, how did you solve it?
